pkg/platform/kube/test/platform_test.go

```diff
@@ -355,6 +355,57 @@ def handler(context, event):
 	suite.DeployFunctionAndRedeploy(createFunctionOptions, afterFirstDeploy, afterSecondDeploy)
 }
 
+func (suite *DeployFunctionTestSuite) TestDeployWithEnvFrom() {
+	functionName := "test-env-from"
+	_, _ = suite.KubeClientSet.CoreV1().Secrets(suite.Namespace).Create(
+		suite.Ctx, &v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{Namespace: suite.Namespace, Name: "test-platform"},
+			StringData: map[string]string{
+				"SECRET_1": "platform",
+				"SECRET_2": "platform"},
+		},
+		metav1.CreateOptions{})
+
+	_, _ = suite.KubeClientSet.CoreV1().Secrets(suite.Namespace).Create(
+		suite.Ctx, &v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{Namespace: suite.Namespace, Name: "test-function"},
+			StringData: map[string]string{
+				"SECRET_2": "function"},
+		},
+		metav1.CreateOptions{})
```

Comment:
Make sure these secrets are deleted after the test is done.
Either in a `defer` function or in the Teardown.